### DISCUSSION TITLE
Could we batch use the special VobSub export menu?

### DISCUSSION THREAD

**Discussion by r-jean-pierre** [SubtitleEdit/subtitleedit#5008] [Q&A]
Dear author,

I was trying to see how can I benefit form the several cool features of the special VobSub / idx menu to render the subs but in a "batch" mode. I was able to do a lot in batch including command line, but the "user defined" export in vobsub is actually only taking some default parameters (while we can do much more in the specific menu).

Is there something I missed or it's a feature which is not (yet?) implemented?

Of course, if such automation exists (or will exist), the goal is to apply the **same** settings (like font size, color, borderstyle etc.) to a bunch of mkv files

Hope to read for some advice

BR

**Comment by niksedk** [comment]
Hi @r-jean-pierre,

In the "Batch convert" window, you can press the "Settings" button:
![image](https://user-images.githubusercontent.com/1261331/116811551-56f8af00-ab4a-11eb-8c98-3f63f87a79bd.png)


Is that what you mean?
